imports/server/hooks/DiscordHooks.ts

```diff
@@ -133,10 +133,7 @@ const DiscordHooks: Hookset = {
         },
       };
 
-      Meteor.defer(() => {
-        // send actual message
-        bot.postMessageToChannel(channel, msg);
-      });
+      void bot.postMessageToChannel(channel, msg);
```

Comment:
Oh, I guess `bot.postMessageToChannel` was already async, so we didn't need to Meteor.defer it, just kick off the Promise and then ignore it (with the `void` here)?

Reply:
Yeah it fires off an HTTP request and then returns a promise - though that might be new since the Meteor `HTTP` library was deprecated in favor of using `fetch`.